lib/resources/package.rb

```diff
@@ -98,10 +98,22 @@ def info(package_name)
       assignment_re: /^\s*([^:]*?)\s*:\s*(.*?)\s*$/,
       multiple_values: false,
     ).params
+    # On some (all?) systems, the linebreak before the vendor line is missing
+    if params['Version'] =~ /\s*Vendor:/
+      v = params['Version'].split(' ')[0]
```

Comment:
@chris-rock Sure: the `sudo` example in my first comment in this pull request is the exact output from a RHEL 6.4 system. Here's another example from a CentOS 5.4 system:

```
Name        : sudo                         Relocations: (not relocatable)\nVersion     : 1.6.9p17                          Vendor: CentOS\nRelease     : 5.el5                         Build Date: Sat 19 Sep 2009 07:02:06 PM PDT\nInstall Date: Mon 10 Jan 2011 03:19:05 AM PST      Build Host: builder16.centos.org\nGroup       : Applications/System           Source RPM: sudo-1.6.9p17-5.el5.src.rpm\nSize        : 467528                           License: BSD\nSignature   : DSA/SHA1, Sat 19 Sep 2009 08:55:01 PM PDT, Key ID a8a447dce8562897\nURL         : http://www.courtesan.com/sudo/\nSummary     : Allows restricted root access for specified users.\nDescription :\nSudo (superuser do) allows a system administrator to give certain\nusers (or groups of users) the ability to run some (or all) commands\nas root while logging all commands and arguments. Sudo operates on a\nper-command basis.  It is not a replacement for the shell.  Features\ninclude: the ability to restrict what commands a user may run on a\nper-host basis, copious logging of each command (providing a clear\naudit trail of who did what), a configurable timeout of the sudo\ncommand, and the ability to use the same configuration file (sudoers)\non many different machines.\n
```